Analytics-CSharp/Segment/Analytics/Compat/Migration.cs

```diff
@@ -0,0 +1,81 @@
+using System.Collections.Generic;
+using Segment.Analytics;
+using Segment.Serialization;
+
+namespace AnalyticsNetMigrationHelper {
+    public class Traits : Dictionary<string, object>
+    {
+    }
+
+    public class Properties  : Dictionary<string, object>
+    {
+    }
+
+    public static class AnalyticsExtensions
+    {
+        public static void Track(this Analytics analytics, string userId, string eventName)
+        {
+            analytics.Track(eventName, new JsonObject() { {"userId", userId} });
+        }
+        public static void Track(this Analytics analytics, string userId, string eventName, IDictionary<string, object> traits)
+        {
+            traits.Add("userId", userId);
+            analytics.Track(eventName, JsonUtility.FromJson<Segment.Serialization.JsonObject>(JsonUtility.ToJson(traits)));
```

Comment:
passing properties should just work 
```suggestion
            analytics.Track(eventName, properties);
```

Reply:
> There is no implicit reference conversion from 'System.Collections.Generic.IDictionary<string, object>' to 'System.Runtime.Serialization.ISerializable'.

The json string/destring is what we'd do internally to any serializable.
Best alternative I can come up with is:
`analytics.Track(eventName, new JsonObject(properties.ToDictionary(x=>x.Key, x=>x.Value as JsonElement)));`